syntaxes/lpc.tmLanguage

```diff
@@ -958,6 +986,79 @@
 					</dict>
 				</array>
 			</dict>
+			<key>dot_access</key>
+			<dict>
+				<key>match</key>
+				<string>[a-zA-Z0-9_]\s*(\.|-&gt;)\s*[a-zA-Z0-9_]</string>
+				<key>captures</key>
+				<dict>
+					<key>1</key>
+					<dict>
+						<key>name</key>
+						<string>punctuation.separator.dot-access.lpc</string>
+					</dict>
+				</dict>
+			</dict>
+			<key>operators</key>
+			<dict>
+				<key>patterns</key>
+				<array>
+					<dict>
+						<key>match</key>
+						<string>\-\-</string>
+						<key>name</key>
+						<string>keyword.operator.decrement.lpc</string>
+					</dict>
+					<dict>
+						<key>match</key>
+						<string>\+\+</string>
+						<key>name</key>
+						<string>keyword.operator.increment.lpc</string>
+					</dict>
+					<dict>
+						<key>match</key>
+						<string>[\+\-\*\/%]\s*=</string>
+						<key>name</key>
+						<string>keyword.operator.assignment.compound.lpc</string>
+					</dict>
+					<dict>
+						<key>match</key>
+						<string>&amp;=|\^=|&lt;&lt;=|&gt;&gt;=|\|=</string>
+						<key>name</key>
+						<string>keyword.operator.assignment.compound.bitwise.lpc</string>
+					</dict>
+					<dict>
+						<key>match</key>
+						<string><<|>></string>
```

Comment:
swap the < and > for &lt; and &gt; to not break the xml

Reply:
aaaaannnnndddd it changed my text. (ampersand)lt; and (ampersand)gt;

Reply:
Yeah, I have that already, it's just not committed yet. ☺️